spec/site_prism/page_objects/page/results/filters/study_type_page.rb

```diff
@@ -0,0 +1,16 @@
+module PageObjects
+  module Page
+    module Results
```

Comment:
If we make this module `ResultFilters` and move it to `spec/site_prism/page_objects/page/result_filters/` we don't have to change `Result` -> `ResultPage` and all is well in the world 😃